deploy/crds/core_v1_storagecluster_crd.yaml

```diff
@@ -55,6 +55,9 @@ spec:
                       storage cluster components. The key specifies component in format of "kind/component",
                       e.g. "deployment/stork" to pass custom annotations to stork deployment. The value is a map of
                       string that contains custom annotation key and value pairs.
+              resources:
+                type: object
+                description: Specifications of storage cluster resources.
```

Comment:
Can you add all the internal nested fields? Otherwise the v1 CRD schema will prune the fields which are not registered.